getting-started.md

```diff
@@ -0,0 +1,64 @@
+# **Getting Started**
+
+> Ok, I have installed Allstar on my account/organization, now what?
+
+By default, Allstar will not take any actions to avoid unwanted noise. To
```

Comment:
"Allstar app installed on your organization" to be more precise.

this part is not needed "to avoid unwanted noise", can just say not take actions on its own.